drivers/mfd/Kconfig.npm2100

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) 2024 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+config MFD_NPM2100
+	bool "nPM2100 PMIC multi-function device driver"
+	default y
+	depends on DT_HAS_NORDIC_NPM2100_ENABLED
+	select I2C
+	help
+	  Enable the Nordic nPM2100 PMIC multi-function device driver
+
+config MFD_NPM2100_LEVEL_IRQ
+	bool "nPM2100 PMIC IRQ level"
+	depends on MFD_NPM2100
+	help
+	  Use level triggering instead of edge triggering when pin interrupt is enabled.
+	  This can reduce power consumption on some platforms, at the expense of irq latency.
```

Comment:
shouldn't this be in DT?

Reply:
Yeah, I was thinking about that, but wasn't sure exactly how to frame it. I'll add it as an enum property or bool then. As far as I understand, the GPIO DT flags doesn't include the interrupt flags